src/web/requests/bill.http

```diff
@@ -7,14 +7,31 @@ Accept: application/json
   "type": 0,
   "country_of_issuing": "AT",
   "city_of_issuing": "Vienna",
-  "issue_date": "2025-01-22",
+  "issue_date": "2025-02-22",
   "maturity_date": "2025-06-22",
-  "payee": "12D3KooWBN48JhNXAKhKxbUVM1YXRrHTfPYxjWiGpcKYGuR77cy3",
-  "payer": "12D3BooWBN48JhNXAKhKxbUVM1YXRrHTfPYxjWiGpcKYGuR77555",
-  "sum": "5000",
-  "currency": "BTC",
+  "payee": "039180c169e5f6d7c579cf1cefa37bffd47a2b389c8125601f4068c87bea795943",
+  "payer": "02c0b2884fb510672feceacdb3678b30b157b017bcb06a9fd44642aae18b522aaf",
```

Comment:
ah, forgot to update those - sorry!
there is no `payer` field in the payload, only `payee` and `drawee`